lms/static/sass/course/layout/_courseware_preview.scss

```diff
@@ -56,6 +56,7 @@ $proctoring-banner-text-size: 14px;
   }
 
   .preview-specific-student-notice {
+    color: $inverse;
```

Comment:
Not sure if I understand how this code change could have caused the change in the screen shots - this code changes text color, but the before/after screenshots show the background color changing?  Am I missing something?